src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeSummaryRepositoryTest.java

```diff
@@ -163,4 +167,32 @@ void should_skip_bulkscan_container() {
                 )
             );
     }
+
+    @Test
+    void should_return_envelopes_received_and_status_summary_by_selected_date_created() {
```

Comment:
There should be tests or other situations:
1) envelopes received at multiple dates
2) containers have no envelopes for the date
3) containers have no rejected envelopes for the date

Reply:
test cases modified to accommodate these.